src/main/scala/com/cognite/sdk/scala/common/login.scala

```diff
@@ -12,14 +14,22 @@ class Login[F[_]](val requestSession: RequestSession[F]) {
   implicit val loginStatusDecoder = deriveDecoder[LoginStatus]
   implicit val dataLoginStatusDecoder = deriveDecoder[DataLoginStatus]
   def status(): F[LoginStatus] =
-    requestSession
-      .sendCdf { request =>
-        request
-          .get(uri"${requestSession.baseUri}/login/status")
-          .response(asJson[DataLoginStatus])
-          .mapResponse {
-            case Left(value) => throw value.error
-            case Right(value) => value.data
-          }
-      }
+    try {
+      requestSession
+        .sendCdf { request =>
+          request
+            .get(uri"${requestSession.baseUri}/login/status")
+            .response(asJson[DataLoginStatus])
+            .mapResponse {
+              case Left(value) => throw value.error
+              case Right(value) => value.data
+            }
+        }
+    } catch {
```

Comment:
in relationships we tend to use attempt(T) so that we do not have to write trys and catches.